.gitignore

```diff
@@ -8,3 +8,4 @@ dist/
 .vscode/
 .scratch/
 .regula-history
+/rego/rules/remediation.yaml
```

Comment:
I'm in favor of committing this file. It would be nice to review changes and know that it's set when our release automation runs.

Reply:
This is actually my preference as well, but i was trying to follow the precedent of swagger.yaml.